packages/create-react-native-library/templates/common/$.circleci/config.yml

```diff
@@ -71,6 +71,49 @@ jobs:
           path: coverage
           destination: coverage
 
+  e2e_release_ios:
```

Comment:
I'd name it something like `e2e-tests-ios`, `release` feels like I'm publishing, also we use `-` in other names as convention